src/ApiService.php

```diff
@@ -48,7 +49,15 @@ public function __construct(
    * {@inheritdoc}
    */
   public function getConfig() {
-    return $this->config;
+    $config = $this->config;
+
+    // Add the Drupal framework details here.
+    $config += [
+      'framework' => 'Drupal',
+      'frameworkVersion' => Drupal::VERSION,
+    ];
+
+    return $config;
```

Comment:
**CHANGE REQUEST**

I still think that the config should be initialized in the constructor, see comment in PR #4. The rest looks good.